packages/admin-stories/src/admin/error-handling/ErrorDialogGraphql.tsx

```diff
@@ -0,0 +1,86 @@
+import { LocalErrorScopeApolloContext, Table, TableQuery, useTableQuery } from "@comet/admin";
+import { Typography } from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import { boolean, text } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import gql from "graphql-tag";
+import * as React from "react";
+
+import { apolloSwapiStoryDecorator } from "../../apollo-swapi-story.decorator";
+import { errorDialogStoryProviderDecorator } from "../../error-dialog-provider.decorator";
+
+interface People {
+    id: string;
+    name: string;
+    birthYear: string;
+    gender: string;
+    homeworld: {
+        name: string;
+    };
+}
+interface QueryData {
+    allPeople: {
+        people: People[];
+    };
+}
+
+const Story: React.FunctionComponent = () => {
+    const query = gql`
+        ${text("GQL Query", "query StarWarsPeople {allPeople { people { id name birthYear gender homeworld{ name } } }}")}
+    `;
+
+    const globalErrorScope = boolean("Use Error Dialog", true);
+
+    const { tableData, api, loading, error } = useTableQuery<QueryData, {}>()(query, {
+        resolveTableData: (data) => ({
+            data: data.allPeople.people,
+            totalCount: data.allPeople.people.length,
+        }),
+        context: globalErrorScope ? undefined : LocalErrorScopeApolloContext,
```

Comment:
shall we implement a variable in the useTableQuery that is automatically set to local, because our table support local error handling?

Reply:
hm, good question. useTableQuery can be used without TableQuery but won't in pratice.

I lean towards setting automatically to local.

Reply:
just added a prop `globalErrorHandling` to useTableQuery with the default value `false`